app/routes/events_.$eventId.tsx

```diff
@@ -0,0 +1,86 @@
+import type { LoaderFunction } from '@remix-run/node';
+import { useParams, useLoaderData, Link } from '@remix-run/react';
+import { db } from '~/modules/database/db.server';
+import { json } from '@remix-run/node';
+import { eventDataPatcher } from '~/modules/events/event';
+
+export const loader: LoaderFunction = async ({ params }) => {
+  const event = await db.event.findFirst({
+    where: { id: params.eventId },
+    include: {
+      group: true,
+      users: {
+        select: {
```

Comment:
We are using the the user's `id` further down in the file, so let's add `id: true` here as well.